extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -50,8 +56,19 @@ class AmpAccordion extends AMP.BaseElement {
       header.setAttribute('role', 'tab');
       content.classList.add('-amp-accordion-content');
       content.setAttribute('role', 'tabpanel');
-      content.setAttribute(
-          'aria-expanded', section.hasAttribute('expanded').toString());
+      if (!accordionSession_) {
+        content.setAttribute(
+            'aria-expanded', section.hasAttribute('expanded').toString());
+      } else {
```

Comment:
this should happen in the bottom of your logic - 

so your code should be 

if (accordionSession_) {
 .......
}

content.setAttribute(
         'aria-expanded', section.hasAttribute('expanded').toString());